### PR TITLE
Add distance measurement tool

### LINKED ISSUE #21: Add Distance measurement tool

### PR DESCRIPTION
**Pull request by spycolyf**
Please review this code. It is my attempt to implement to Distance Measurement Tool. There is a known issue with the edit function that replaces the QREADS tool bar. The absolute best solution would be to allow the user to get the QREADS toolbar back rather than eliminating the very useful Slicer measurement line edit function.

### CODE REVIEW

File: Modules/Scripted/QReads/QReads.py [hunk 1]
```diff
@@ -275,6 +276,18 @@ def showSlicerQREADSHelp(self):
       self.helpWebWidget.url = qt.QUrl('http://www.google.com')
     self.helpWebWidget.show()
 
+  def createDistMeasTool(self):
```
Comment (jcfr) [inline Modules/Scripted/QReads/QReads.py]: No need to abbreviate, consider using fully descriptive name

File: Modules/Scripted/QReads/QReads.py [hunk 2]
```diff
@@ -275,6 +276,18 @@ def showSlicerQREADSHelp(self):
...
+    # Create the SlicerQREADS Orientation Marker Type
+    lineNode = slicer.mrmlScene.CreateNodeByClass("vtkMRMLMarkupsLineNode")
+    lineNode.SetName("Line")
+    slicer.mrmlScene.AddNode(lineNode)
```
Comment (jcfr) [inline Modules/Scripted/QReads/QReads.py:305]:
```python
lineNode.UnRegister(slicer.mrmlScene)
```

See https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MemoryManagement#Python_scripts_and_scripted_modules

**Review by jcfr — CHANGES_REQUESTED** [review]
Consider rebasing the changes and addressing comments

**Comment by jcfr** [comment]
Only the relevant actions are visible when right-clicking:

image

**Review by jcfr — APPROVED** [review]

**Comment by spycolyf** [comment]
I'm currently making the following changes:

1) Make the Distance measurement Tool Button a toggle button.
2) When toggle is checked, enable 1 drawing on any image
3) When toggle is unchecked, Delete line.

**Comment by jcfr** [comment]
> I'm currently making the following changes:

Consider creating a new issue and new branch based on the latest version. Thanks

**Comment by jcfr** [comment]
> When toggle is unchecked, Delete line.

This has already been implemented using right click.

**Comment by spycolyf** [comment]
OK, Could you make the Interactive handles also a a right mouse choice if easy to do

**Comment by jcfr** [comment]
> OK, Could you make the Interactive handles also a a right mouse choice if easy to do

Consider creating an issue and associated the "Backlog" milestone. For now, I consider this is a very minor tweak that should be addressed at later stage after allocating resources for this project.

**Comment by spycolyf** [comment]
OK

**Pull request merged**